docs/en/04_Changelogs/3.7.0.md

```diff
@@ -0,0 +1,117 @@
+# 3.7.0
+
+## HTTP Cache Header changes
+
+### Overview
+
+In order to support developers in making safe choices around HTTP caching,
+we've introduced a `HTTPCacheControl` class to control if a response
+should be considered public or private. This is an abstraction on existing
+lowlevel APIs like `HTTP::add_cache_headers()` and `SS_HTTPResponse->addHeader()`.
+
+This change introduces smaller but necessary changes to HTTP caching headers
+sent by SilverStripe. If you are relying on HTTP caching in your implementation,
+or use modules such as [silverstripe/controllerpolicy](https://github.com/silverstripe/silverstripe-controllerpolicy),
+please review the implications of these changes below. 
+
+In short, these APIs make it easier to express your caching preferences
+without running the risk of overriding essential core safety measures.
+Most commonly, these APIs will prevent HTTP caching of draft content.
+
+It will also prevent content generated with an active session,
+since the system can't tell whether session data was used to vary the output.
+In this case, it's up to the developer to opt-in to caching,
+after ensuring that certain execution paths are safe despite of using sessions.
+
+The system behaviour does not guard against accidentally caching "private" content,
+since there are too many variations under which output could be considered private
+(e.g. a custom "approval" flag on a comment object). It is up to
+the developer to ensure caching is used appropriately there.
+
+By default, SilverStripe sends headers which signal to HTTP caches
+that the response should be considered not cacheable.
+
+See [Developer Guide: Performance > HTTP Cache Headers](/developer_guide/performance/http_cache_headers)
+for details on the new API.
+
+### Example Usage
+
+Globally opting into HTTP caching for all page content.
+SilverStripe will still override this preference when a session is active,
+or draft content has been requested. 
+
+```diff
+class Page_Controller extends ContentController
+{
+    public function init()
+    {
+-        HTTP::set_cache_age(60);
++        HTTPCacheControl::inst()
++           ->publicCache()
++           ->setMaxAge(60); // 1 minute
+
+        
+        parent::init();
+    }
+}
+```
+
+Opting out of HTTP caching in a particular controller action.
+
+```diff
+class MyPage_Controller extends Page_Controller
+{
+    public function myprivateaction($request)
+    {
+        $response = $this->myPrivateResponse();
+-        HTTP::set_cache_age(0);
++        HTTPCacheControl::inst()
++           ->privateCache();
```

Comment:
Again, there's a nuance here, but this isn't the same as disabling or "opting out" of http caching